packages/map3d/src/interfaces/planet.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface PlanetInterface {
+  uuid: string;
+  owner: string;
+  name: string;
+  image?: string | undefined;
```

Comment:
```suggestion
  image?: string;
```

why repeat yourself? :)

Reply:
It is two different cases. There is no field at all and prop is undefined.

Reply:
but it doesn't stop you from setting it as undefined explicitly 😊

```
image?: string;
```
is different from 
``` 
image: string | undefined
```
but not from 
```
image?: string | undefined;
```